GetUpChuckUnityProject/Assets/Scripts/Game World/Pit.cs

```diff
@@ -0,0 +1,25 @@
+﻿using UnityEngine;
+using System.Collections;
+
+public class Pit : Obstacle, MonoBehaviour {
+
+	bool inPit = false;
+	
+	void OnCollisionEnter2D(Collision2D coll) {
+		if (coll.gameObject.tag == "Player") { condCheck(); }
```

Comment:
use 
coll.gameObject.tag.Equals("Key")
for string comparison

condCheck needs to be passed an inventory